classes/local/table/interaction_remaining_table.php

```diff
@@ -73,6 +73,8 @@ public function __construct($uniqueid, $courseids) {
                 'INNER JOIN ( ' .
                     'SELECT b.courseid as cid, MAX(b.id) as maxlogid ' .
                     'FROM {tool_lifecycle_action_log} b ' .
+                    'INNER JOIN {tool_lifecycle_workflow} w ON b.workflowid = w.id ' .
+                    'WHERE w.timeactive IS NOT NULL ' .
```

Comment:
Processes which have been stopped by the hand sign and not the stop sign might still have running courses which should be displayed although timeactive is null. Therefore, we need a way to distinguish between workflows stopped by the hand and workflows stopped by the stop sign.
The class process manager checks `$DB->count_records('tool_lifecycle_process', array('workflowid' => $workflowid));` for differentiating between workflows which can be deleted and workflows with active processes. This might be some way to work around it.

Reply:
To be clear do I check all workflows for active processes or only the deactivated ones and display all active ones regardless of active processes?

Reply:
I am not sure but my latest commit might have fixed the issue.